spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -116,6 +116,8 @@ module MiqAeServiceSpec
           Datacenter
           ManageIQ::Providers::BaseManager
           ManageIQ::Providers::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::Provider
```

Comment:
@juliancheal @gmcculloug I can't tell if you're both "fine" with excluding these for now or if you're agreeing that the service models should be added in this PR.